src/libs/OptionsListUtils.js

```diff
@@ -412,17 +410,9 @@ function getOptions(reports, personalDetails, draftComments, activeReportID, {
     // Always exclude already selected options and the currently logged in user
     const loginOptionsToExclude = [...selectedOptions, {login: currentUserLogin}];
 
-    if (excludeConcierge) {
-        loginOptionsToExclude.push({login: CONST.EMAIL.CONCIERGE});
-    }
-
-    if (excludeChronos) {
-        loginOptionsToExclude.push({login: CONST.EMAIL.CHRONOS});
-    }
-
-    if (excludeReceipts) {
-        loginOptionsToExclude.push({login: CONST.EMAIL.RECEIPTS});
-    }
+    _.each(excludeLogins, (excludeLogin) => {
+        loginOptionsToExclude.push({login: excludeLogin});
+    });
```

Comment:
`excludeLogin` is kinda weird here, we can just use `login`.

```suggestion
    _.each(excludeLogins, (login) => {
        loginOptionsToExclude.push({login});
    });
```